style.css

```diff
@@ -302,3 +302,69 @@ button:disabled {
   border-radius: 0;
   color: #000;
 }
+
+.contacts {
+  background-image: url("./images/contactshapes.png");
+  display: flex;
+  background-color: #396df2;
```

Comment:
- [ ] Great work Implementing the project requirements :clap: , however as one of the main goal of the project is to parse a figma template as close to the original design as possible, please use the correct background color for this section it should be ` background-color: #6070FF;`